---
title: "PTM-Shepherd: analysis and summarization of post-translational and chemical modifications
from open search results"
collection: publications
permalink: /publications/2020-12-01-ptm-shepherd-analysis-and-summarization
excerpt: '<i>Open searching has proven to be an effective strategy for identifying both known and unknown modifications in shotgun proteomics experiments. Rather than being limited to a small set of userspecified modifications, open searches identify peptides with any mass shift that may correspond to a single modification or a combination of several modifications. Here we present PTMShepherd, a bioinformatics tool that automates characterization of PTM profiles detected in open searches based on attributes such as amino acid localization, fragmentation spectra similarity, retention time shifts, and relative modification rates. PTM-Shepherd can also perform multiexperiment comparisons for studying changes in modification profiles, e.g. in data generated in different laboratories or under different conditions. We demonstrate how PTM-Shepherd improves the analysis of data from formalin-fixed paraffin-embedded samples, detects extreme underalkylation of cysteine in some datasets, discovers an artefactual modification introduced during peptide synthesis, and uncovers site-specific biases in sample preparation artifacts in a multi-center proteomics profiling study.</i>'
date: 2020-12-01
venue: 'Molecular and Cellular Proteomics'
paperurl: 'https://doi.org/10.1074/mcp.TIR120.002216'
citation: '<b>Daniel J. Geiszler</b>, Andy T. Kong, Dmitry M. Avtonomov, Fengchao Yu, Felipe da Veiga Leprevost, Alexey I. Nesvizhskii. (2020). &quot;PTM-Shepherd: analysis and summarization of post-translational and chemical modifications from open search results.&quot; <i>Mol. Cell Proteomics</i>. In press.'
---
<i>Open searching has proven to be an effective strategy for identifying both known and unknown
modifications in shotgun proteomics experiments. Rather than being limited to a small set of userspecified modifications, open searches identify peptides with any mass shift that may correspond to a single modification or a combination of several modifications. Here we present PTMShepherd, a bioinformatics tool that automates characterization of PTM profiles detected in open searches based on attributes such as amino acid localization, fragmentation spectra similarity, retention time shifts, and relative modification rates. PTM-Shepherd can also perform multiexperiment comparisons for studying changes in modification profiles, e.g. in data generated in different laboratories or under different conditions. We demonstrate how PTM-Shepherd improves the analysis of data from formalin-fixed paraffin-embedded samples, detects extreme underalkylation of cysteine in some datasets, discovers an artefactual modification introduced during peptide synthesis, and uncovers site-specific biases in sample preparation artifacts in a multi-center proteomics profiling study.</i>

[Download paper here](https://www.mcponline.org/content/mcprot/early/2020/12/01/mcp.TIR120.002216.full.pdf)
